resources/views/themes/bimbala/layouts/app.blade.php

```diff
@@ -57,6 +57,40 @@
     @include('theme::partials.header')
 
     <main class="flex-grow overflow-x-hidden">
+        @unless (Request::path() == '/' ||  Request::path() == 'login'||  Request::path() == 'pricing' || Request::path() == 'register' ||Request::path() == 'dashboard' || Request::is('account/*')|| Request::is('password/*') || Request::is('@*') || Request::is('blog/*'))
+            <div class="bg-gray-900">
+                <div class="max-w-7xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:px-8 lg:flex lg:justify-between">
+                    <div class="max-w-xl">
+                        <h2 class="text-4xl font-extrabold text-white sm:text-5xl sm:tracking-tight lg:text-6xl">{{ $seo['seo_title']}}</h2>
+                        <p class="mt-5 text-xl text-gray-300">{{ $seo['seo_description'] }}</p>
+                    </div>
+                    @if(Request::path() == 'blog' || Request::path() == 'use-case' || Request::path() == 'about-us')
+                    <div class="mt-10 w-full max-w-xs">
```

Comment:
This looks good on mobile, but not on desktop:
![image](https://user-images.githubusercontent.com/14351652/179608312-12cd4acf-48ec-4ccf-98db-955a144a255d.png)

Reply:
mhm, have you tried rebuilding the css? - for me it looks fine on both (mobile and desktop)

Reply:
Yes, after rebuilding the css works fine. Let's upload the changed views along with the rebuilt assets.